Patch to allow building libffi using clang:
  https://github.com/libffi/libffi/issues/191

Can be dropped after the next libffi release.

diff -u -r ../libffi-3.2.1/src/arm/sysv.S ./src/arm/sysv.S
--- ../libffi-3.2.1/src/arm/sysv.S	2014-11-08 07:47:24.000000000 -0500
+++ ./src/arm/sysv.S	2016-05-16 20:37:25.684302345 -0400
@@ -396,7 +396,7 @@
 	beq	LSYM(Lepilogue_vfp)
 
 	cmp	r3, #FFI_TYPE_SINT64
-	stmeqia	r2, {r0, r1}
+	stmiaeq	r2, {r0, r1}
 	beq	LSYM(Lepilogue_vfp)
 
 	cmp	r3, #FFI_TYPE_FLOAT
